website/reference/all-config.md

```diff
@@ -2,6 +2,8 @@
 
 ## `monorepo`
 
+_type_: `object`
```

Comment:
for example:

```suggestion
_type_: `{ monorepo: { mainVersionFile: string, packagesToBump: string[], packagesToPublish: string[], updateDependencies?: boolean } }`
```

although I think being combined with the example could make sense

Reply:
```
import { defineConfig } from 'vite'

export default defineConfig({
  // ...
})
```

[vite](https://vitejs.dev/config/#config-intellisense) supports the syntax like that which enables intellisense.

We could go with Haroen's approach with all the parameters explained in the documentation, or write a proper typing for better configuring on the IDE level.

However just having `object` in the documentation doesn't seem to add much value to me.